package.json

```diff
@@ -71,6 +70,7 @@
     "react": "^16.12.0",
     "react-docgen-typescript-loader": "^3.6.0",
     "react-dom": "^16.12.0",
+    "react-is": "^17.0.1",
```

Comment:
I can't find this to be referenced in this PR. is it a necessary peer depencendy or an oversight?

Reply:
How can I check it this is a necessary peer dependency ? Can I be sure by checking the "requires" sections in package-lock ?

Reply:
you can delete the line and run `npm install` - this would warn you about missing peer dependencies if it's required by any package that we're using. if you didn't add it intentionally, it might very well be that it slipped in somehow

Reply:
I checked, it is a peer dependency for addon-docs and addon-essential